components/main_menu/elements/icon_button_common.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+interface Props {
+    iconGlyph: string;
+    size: string;
+    label: string;
+    disabled?: boolean;
+    additionalClass?: string;
```

Comment:
0/5 on this naming, but maybe something like `buttonClassName` would make sense here? Saying what is being styled might be a good idea in case we end up having custom components that support multiple additional classes. I suppose we can always change that if we run into that being a problem, but we'll want to be consistent with whatever we pick.

Reply:
I agree with you, we should decide on a unified naming. My thinking was that when we render the components it will be clear what the element name is and then it's clear what the additionalClass represents. Like `<IconButtonCommon additionalClass='happy'/>` or `<IconCommon additionalClass='sad'/>`

Reply:
As long as these components stay small, that should work, although if they start to contain more, then it might be necessary to be more specific. Like maybe we have a `List` component that needs something like `containerClassName` and `listItemClassName`.

Also, I'm leaning towards calling them `additionalClassName` over `additionalClass` just because that's what React uses elsewhere